utils/dataset_manifest/README.md

```diff
@@ -0,0 +1,36 @@
+## Simple command line for prepare dataset manifest file
```

Comment:
Some installation instructions could be helpful, maybe a `requirements.txt`, because `av` and `cv2` are used.